Towerise/ClientApp/app/components/play-space/play-space.component.ts

```diff
@@ -52,6 +56,14 @@ export class PlaySpaceComponent implements OnInit {
             this.gridLines.push({ x1: -500, x2: 500, y1: y * 100, y2: y * 100 });
         }
 
+
+        for (let x = -5; x < 5; x++) {
```

Comment:
what's the significance of these numbers? why -5 to 5, and why iterating 10 times on each loop?

Reply:
this adds the coordinate text, to the center of each square. Visual only.

-5 to 5 because I didn't want to offset the the coordinate inside the loop by -500